src/routes/safe/components/Settings/ManageOwners/RemoveOwnerModal/screens/Review/index.tsx

```diff
@@ -87,9 +88,9 @@ export const ReviewRemoveOwnerModal = ({
       try {
         const gnosisSafe = await getGnosisSafeInstanceAt(safeAddress)
         const safeOwners = await gnosisSafe.methods.getOwners().call()
-        const index = safeOwners.findIndex((owner) => owner.toLowerCase() === ownerAddress.toLowerCase())
+        const index = safeOwners.findIndex((owner) => sameAddress(owner, ownerAddress))
         const prevAddress = index === 0 ? SENTINEL_ADDRESS : safeOwners[index - 1]
-        const txData = gnosisSafe.methods.removeOwner(prevAddress, ownerAddress, threshold).encodeABI()
+        const txData = gnosisSafe.methods.removeOwner(prevAddress, ownerAddress, threshold - 1).encodeABI()
```

Comment:
Why do threshold need to be modified here? Threshold is supposed to be set in previous step. So if threshold is set to 1 we would be trying to set threshold to 0

I think this change is dangerous as a last minute fix

Reply:
@dasanra good catch. It's fixed, can you check now?